### DISCUSSION TITLE
How to use ImportC?

### DISCUSSION THREAD

**Discussion by ghost** [dlang/dmd#13547] [Q&A] [answered]
I noticed there's close to no documentation for ImportC, and the tests in the repo don't work. How would I use ImportC in a program? Is it completely usable yet?

**Comment by MoonlightSentinel** [comment, marked as answer]
Reports for missing documentation should go to [bugzilla](https://issues.dlang.org/). The discussions are more targeted at DMD's internals,....

> [...]  and the tests in the repo don't work.

What command did you use and what was the failure?

**Reply by ghost** [reply]
Oh, my bad. I didn't run the specific test, instead I copied it, and attempted to compile it with ldc and dmd using the shell file as a reference, which might be the problem. I lost the errors, I believe they were syntax errors, but I can try and replicate the error again.

**Reply by MoonlightSentinel** [reply]
Those test files probably require changes that were merged but not yet included in a release.
You might be interested in the [nightly build](https://github.com/dlang/dmd/releases/tag/nightly).

**Reply by ghost** [reply]
Alright, thanks

**Reply by ghost** [reply]
Yep, it successfully compiles and runs now, thanks so much! Still not sure if it should be used instead of making bindings yet though...
